pkg/kubectl/cmd/top/top_pod_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cmd
+package top
```

Comment:
@mengqiy I checked that `set` package also has `set_` prefix, do you think that also should be removed as well ?